seekinternational/notification-library
# Tech task/upgrade node 8.11.4

What does this PR do?
---------------------
To update nodeJs to latest LTS version which is 8.11.4. 
This changes include:
- refactoring callback with async await.
- Remove unnecessary babel configuration

How can I test this?
--------------------
run `npm test`

Notes
-----
Need to ensure everyone that have this package is installed with release tag as this will break all the repositories that is installed using the master version.
